integration-tests/src/test/java/uk/gov/di/authentication/services/DynamoServiceIntegrationTest.java

```diff
@@ -1,11 +1,15 @@
 package uk.gov.di.authentication.services;
 
 import com.nimbusds.oauth2.sdk.id.Subject;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
Minor typo in the commit message for this change.  (oof vs of)

Reply:
Good spot, fixed!